capnpy/segment/builder.pyx

```diff
@@ -12,6 +11,12 @@ cdef extern from "Python.h":
     int PyByteArray_Resize(object o, Py_ssize_t len)
     char* PyByteArray_AS_STRING(object o)
 
+cdef extern from "_util.h":
```

Comment:
I have tried that and even when using only `cdef` and `cimport` and even `inline` there was always a slight slowdown in `ctor` benchmarks. However, there might be a trick to that I am not aware of.